resolwe_bio/processes/workflows/wgs_gvcf.py

```diff
@@ -359,17 +371,16 @@ def run(self, inputs, outputs):
                 "assume_sorted": True,
             },
         )
-
+        multiqc_inputs = [
+            inputs.reads,
+            bam,
+            summary,
+            wgs_metrics,
+            insert_size,
+        ]
+        if inputs.trimming_options.enable_trimming:
+            multiqc_inputs.append("trimmomatic")
```

Comment:
`trimmomatic` should be a variable and not a string.

Reply:
Fixed, thank you!